### ISSUE TITLE
Multihttp & Scripted check assertions - success rate by probe graph doesn't show failures properly

### ISSUE THREAD

**Issue by peterschretlen** [grafana/synthetic-monitoring-app#786]
**What happened**:
Assertions in a multihttp and/or scripted check are failing, but the chart does not show when the failures occur. 

Here is an example of an assertion that failed 58 times over the course of a week.  The success chart shows some gaps and some dots, but it never goes below 100%.  This makes it very hard to narrow down when/where the failures occurred.
<img width="1564" alt="image" src="https://github.com/grafana/synthetic-monitoring-app/assets/7903188/e189fa46-207b-441e-a071-25ff0bef8201">

**What you expected to happen**:

If I open explore from the dashboard panel I can see that the interval is every 2m,  for a check that runs on a 2m interval. 
When I change the interval to 5m, I can now see where the failures occured

<img width="1590" alt="image" src="https://github.com/grafana/synthetic-monitoring-app/assets/7903188/e54e30aa-9b10-4aea-9f09-81963e9d1733">

**How to reproduce it (as minimally and precisely as possible)**:

This script will randomly fail about 10% of the time.  Run this as a scripted check, and look at the assertion graph for '2. is result < 10'.  The plot lines will never go below 100% even though it is failing. 
```
import { check, fail } from 'k6'
import http from 'k6/http'

export default function main() {
  const res = http.get('https://www.randomnumberapi.com/api/v1.0/random?min=1&max=10');
  // console.log will be represented as logs in Loki
  console.log(`response ${res.json('[0]')}, ${res.status}`);
  check(res, {
    '1. is status 200': (r) => r.status === 200,
    '2. is result < 9': (r) => parseInt(res.json('[0]')) <  9,
  }) || fail(`check failed`);;
}
```

**Comment by peterschretlen** [comment]
I think an attempt was made to fix it here (the panel used to have a fixed interval of 5m)  https://github.com/grafana/synthetic-monitoring-app/pull/772/files, but the interval still seems to low?

**Comment by peterschretlen** [comment]
This might be partly related to https://github.com/grafana/synthetic-monitoring/issues/55.  Since we have gaps in the logs, we also show gaps in the charts.  Even if the logs are fixed though, I think showing the failures might still be an issue

**Comment by peterschretlen** [comment]
https://github.com/grafana/synthetic-monitoring/issues/55 should be fixed after v0.53 of k6 is released.  That will help this problem somewhat.

**Comment by VikaCep** [comment]
As you correctly mentioned @peterschretlen, the visualization looks much better if we increase the interval, as right now it calculates it so it's the same as the check's frequency (2m in the example)

- [2m]

![image](https://github.com/user-attachments/assets/7c16c717-6e5e-48d3-830f-9ea5f6431245)


- Setting it to [3m]:

![image](https://github.com/user-attachments/assets/d5254b1f-a18f-4965-a582-740ea9005995)

- Setting it to [5m]:

![image](https://github.com/user-attachments/assets/00d18717-d7e2-414d-b685-c8e2ca9d5e93)

We could maintain the calculation for the `minStep` based on the frequency of the check, but add a few minutes to it which seems to fix showing the failures, even though there are still some gaps in the visualization. I'm not sure whether just 1 minute extra is a good solution or should we calculate it based on a different criteria though. Do you have any opinions on this?
